kie-server-parent/kie-server-common/src/main/java/org/kie/server/common/rest/RestEasy960Util.java

```diff
@@ -37,12 +38,14 @@
     private static final String ACCEPT_LANGUAGE = "Accept-Language";
    
     public static List<Variant> variants 
-        = Variant.mediaTypes(MediaType.APPLICATION_XML_TYPE, MediaType.APPLICATION_JSON_TYPE).add().build();
+            = Variant.mediaTypes(MediaType.APPLICATION_XML_TYPE, MediaType.APPLICATION_JSON_TYPE,
+                                 new MediaType("application", "json", Collections.singletonMap("strict", "true")),
+                                 new MediaType("application", "xml", Collections.singletonMap("stritct", "true"))).add().build();
```

Comment:
Just one note that there was a typo in `stritct`. But I guess you tried it locally without it. It's a pity that it didn't work. What about my idea to do it directly in our marshaller?

Reply:
@MarianMacik #1907 the is done in the JSONMarshaller, it is the only way I found to keep backward compatibility.